src/compiler/scala/tools/nsc/ast/parser/Parsers.scala

```diff
@@ -1532,8 +1528,8 @@ self =>
               }
             }
           val finalizer = in.token match {
-            case FINALLY => in.nextToken(); expr()
-            case _ => EmptyTree
+            case FINALLY => in.nextToken() ; expr()
+            case _       => EmptyTree
```

Comment:
We don't need scalafmt, just somfmt.

Reply:
I'd changed it to if-else in the earlier patch, but now I always prefer match. This is how I typed it back in.